### PR TITLE
fix(plugin-meetings): imports of MediaConnection from internal-media-core

### PR DESCRIPTION
**Pull request by marcin-bazyl**
## This pull request addresses

TS errors related to MediaConnection types.

## by making the following changes

Fix is done in @webex/internal-media-core lib in this [PR](https://sqbu-github.cisco.com/WebExSquared/webrtc-media-core/pull/157) and this SDK PR is just updating the SDK accordingly so that it builds: basically we now import all the MediaConnection types directly from @webex/internal-media-core, they are not wrapped with `MediaConnection` anymore.

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

unit tests

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [x] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/plugin-meetings/package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
   },
   "dependencies": {
     "@webex/common": "workspace:^",
-    "@webex/internal-media-core": "^0.0.17-beta",
+    "@webex/internal-media-core": "1.32.1",
```
Comment (torgeadelin) [inline packages/@webex/plugin-meetings/package.json:45]: was this intentional?

**Review by torgeadelin — APPROVED** [review]
LGTM, just a small question

**Review by Coread — APPROVED** [review]

File: packages/@webex/plugin-meetings/package.json (hunk 1 above)
Comment (mkesavan13) [reply packages/@webex/plugin-meetings/package.json:45]: I am also curious. Because this looks like a version that'll get published to internal jFrog env.

Comment (arun3528) [reply packages/@webex/plugin-meetings/package.json:45]: ya the main error was the media-core was not exporting the module properly

**Review by arun3528 — APPROVED** [review]

**Pull request merged**